app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionsViewModel.java

```diff
@@ -56,6 +60,8 @@ public void destroy() {
                     this.rooms = extractRooms(sessions);
                     this.stimes = extractStimes(sessions);
 
+                    notifyPropertyChanged(BR.loadingVisibility);
```

Comment:
I think it works well. But I prefer Rx way like this.
http://qiita.com/boohbah/items/e8010730725c54f85a3a

What do you think? Just I want to know your opinion 😃